### PR TITLE
Efficient gene labels

### PR DESCRIPTION
**Pull request by mathieupost**
Much efficiency!

### CODE REVIEW

**Comment by TomBrouws** [comment]
Still gives a  java.lang.OutOfMemoryError: GC overhead limit exceeded on 671 strains.

**Comment by mathieupost** [comment]
But still is much faster than before so I think we should still merge this PR.
Will look at the OOM in my next PR.

File: src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java [hunk 1]
```diff
@@ -42,12 +42,12 @@
 	/**
 	 * Text that is displayed in the About Me option in the Help menu.
 	 */
-	final String about = "Helix� is an interactive DNA sequence viewer. "
+	final String about = "Helix� is an interactive DNA sequence viewer. "
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java]: Unknown character? Create a util class to collect all special strings like the program name and use chars to add these characters.

File: pom.xml [hunk 2]
```diff
@@ -21,11 +24,6 @@
     </dependency>
     <dependency>
       <groupId>org.graphstream</groupId>
-      <artifactId>gs-core</artifactId>
-      <version>1.2</version>
-    </dependency>
-    <dependency>
-      <groupId>org.graphstream</groupId>
```
Comment (Delfil) [inline pom.xml]: The gs-core is included somewhere else?

**Comment by Delfil** [comment]
I'm able to load 100 strains graphs again!
Everything works & code is fine.

**Comment by kwendel** [comment]
looks good

File: pom.xml (hunk 2 above)
Comment (mathieupost) [inline pom.xml]: It was defined two times. See the dependency under it.

**Pull request merged**